scene/resources/convex_polygon_shape_2d.cpp

```diff
@@ -65,7 +65,7 @@ bool is_convex(const Vector<Vector2> &p_points) {
 
 void ConvexPolygonShape2D::_update_shape() {
 	Vector<Vector2> final_points = points;
-	if (Geometry2D::is_polygon_clockwise(final_points)) { //needs to be counter clockwise
+	if (!Geometry2D::is_polygon_clockwise(final_points)) { //needs to be clockwise
```

Comment:
```suggestion
	if (!Geometry2D::is_polygon_clockwise(final_points)) { // Needs to be clockwise.
```
Just a minor comment style nitpick

Reply:
Addressed in 1690c03d0d4b09ddd75853c8f5a44dbfd1d91a3c.